pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -310,7 +313,30 @@ func TestSyncMurStatusWithUserAccountStatusWhenCompleted(t *testing.T) {
 
 		// then
 		require.NoError(t, err)
-		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, toBeProvisioned(), toBeProvisionedNotificationCreated())
+		expectedProvisionedTime := metav1.Now()
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, expectedProvisionedTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
+	})
+
+	t.Run("ProvisionedTime should not be updated when synced more than once", func(t *testing.T) {
+		// given
+		hostClient := test.NewFakeClient(t, mur)
+		sync, memberClient := prepareSynchronizer(t, userAccount, mur, hostClient)
+
+		// when
+		err1 := sync.synchronizeStatus() // 1st sync should not change anything, including ProvisionedTime in MUR
+		initialProvisionedTime := sync.record.Status.ProvisionedTime
+
+		expectedProvisionedTime := metav1.Now()
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, expectedProvisionedTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
+
+		// mock changed MUR before syncing again
+		murtest.Modify(mur, murtest.StatusCondition(toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, "")))
+		err2 := sync.synchronizeStatus() // 2nd sync should update the status but not the ProvisionedTime in the MUR
+
+		// then
+		require.NoError(t, err1)
+		require.NoError(t, err2)
+		require.True(t, initialProvisionedTime.Time.Equal(sync.record.Status.ProvisionedTime.Time)) // timestamp should be the same
```

Comment:
couldn't we make the whole test much simpler like this?:
```suggestion
		// given
		hostClient := test.NewFakeClient(t, mur)
		sync, memberClient := prepareSynchronizer(t, userAccount, mur, hostClient)
		provisionTime := metav1.NewTime(time.Now().Add(-time.Hour))
		sync.record.Status.ProvisionedTime = &provisionTime

		// when
		err := sync.synchronizeStatus()

		// then
		require.NoError(t, err)
		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, provisionTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
		assert.Equal(t, provisionTime.Time, sync.record.Status.ProvisionedTime.Time) // timestamp should be the same
```

Reply:
Yes, this is much simpler. 🙂  Thanks for the suggestion!